crates/sui-storage/benches/event_store_bench.rs

```diff
@@ -4,21 +4,42 @@
 #[macro_use]
 extern crate criterion;
 
+use std::sync::Mutex;
+
 use criterion::{BenchmarkId, Criterion, Throughput};
+use lazy_static::lazy_static;
 use tempfile::NamedTempFile;
 use tokio::runtime::Builder;
 
 use sui_storage::event_store::{sql::SqlEventStore, test_utils, EventStore};
 use sui_types::{
-    base_types::SuiAddress,
+    base_types::{SuiAddress, TransactionDigest},
     event::{EventEnvelope, TransferType},
 };
 
 async fn repeat_batch_insert(db: &SqlEventStore, events: &[EventEnvelope], batch_size: usize) {
-    // Reset sequence number so we can insert events with old sequence numbers
-    db.testing_only_reset_seq_num();
-    for chunk in events.chunks(batch_size) {
-        db.add_events(chunk).await.expect("Inserts should not fail");
+    lazy_static! {
+        static ref NEXT_SEQ: Mutex<u64> = Mutex::new(0);
+    }
+
+    let mut events: Vec<EventEnvelope> = events.to_vec();
+
+    let mut seq: u64 = {
+        let mut seq = NEXT_SEQ.lock().unwrap();
```

Comment:
Can we call the inner seq something different?  Slightly confusing looking at that code vs the seq used outside